### DISCUSSION TITLE
HTML inside a Datatable Cell?

### DISCUSSION THREAD

**Discussion by nicknunk** [EvotecIT/PSWriteHTML#355] [General]
I just came across PSWriteHTML recently. Great tool so far. 

The one issue I've run into is based around some reports that my team generates with various server related information, dumping them out to .csv for other departments to consume and use. We also use the reports by ingesting the .csv and presenting them as static HTML pages. The thing I can't seem to overcome is based around HTML links that we use in some of the cells - EXAMPLE: inside a cell would have <a href="https://somesite.somewhere.com">Link Name<a/>. 

Using the NewHTML and a Datatable to output the info, this incorrectly presents the HTML in the final output. It sends the links out as text just as presented in the .csv, but not in a format that the final html page is correctly parsing. I've attempted some of the things shown [here](https://github.com/EvotecIT/PSWriteHTML/blob/master/Examples/Example-Texts/Example-Texts01.ps1) and some of the other examples, but the it's the same result. 

I think it has something to do with the way that the "<" and ">" characters are getting interpreted.

Is there a way to escape this, or flag it somehow so that the browser interprets the cells with pure HTML correctly and presents the link correctly in the final output?

Thanks for the consideration! :-)

**Comment by PrzemyslawKlys** [comment]
You need to add `New-HTMLTable -InvokeHTMLTags` switch and it will "activate" HTML within cells.

**Reply by nicknunk** [reply]
Awesome! I will try this. I guess I missed that somewhere. :-)
